### ISSUE TITLE
librewolf-wayland is not building after most recent unstable package refresh

### ISSUE THREAD

**Issue by MetalNeo** [NixOS/nixpkgs#172415]
### Steps To Reproduce
1. upgrade `librewolf-wayland`using `sudo nixos-rebuild switch --upgrade`with package in configuration.nix

### Build log
```
error: builder for '/nix/store/jnr69pl8m1b7a2sgf1lxr3wmydn4ivn1-librewolf-100.0-2.drv' failed with exit code 1
error: 1 dependencies of derivation '/nix/store/7l5l1pdxn9995f5nlmy5cl16b9i5a2vp-system-path.drv' failed to build
copying path '/nix/store/1v5vpif2l238whsl30x6ijwlchfzfy06-gnome-session-42.0' from 'https://cache.nixos.org'...
error: 1 dependencies of derivation '/nix/store/i07rsq4v619x0rq5bsn0pvwx5hgl57wh-nixos-system-nixos-22.05pre376328.2a3aac479ca.drv' failed to build

```

### Additional context
Using unstable package listings

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
this path will be fetched (0.06 MiB download, 0.30 MiB unpacked):
  /nix/store/hk2mydhxsxbw7xalwa4s0dyk0c6gmgkf-bash-interactive-5.1-p16-dev
copying path '/nix/store/hk2mydhxsxbw7xalwa4s0dyk0c6gmgkf-bash-interactive-5.1-p16-dev' from 'https://cache.nixos.org'...
 - system: `"x86_64-linux"`
 - host os: `Linux 5.17.5-zen1, NixOS, 22.05 (Quokka), 22.05pre375061.c777cdf5c56`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.8.0`
 - channels(root): `"nixos"`
 - channels(metalneo): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by MetalNeo** [comment]
Looks like issue may be resolved by pending PR #172358.

**Comment by squalus** [comment]
> Looks like issue may be resolved by pending PR #172358.

Possibly. I did fix a build error in there. It looks like the actual build error is cut off in the log you posted. If you have more of the preceding log lines, that would be helpful.

**Comment by MetalNeo** [comment]
The preceding lines are just relating to fetching/building other packages and system functions. Running with --verbose didn't cause the output to be any different either.

**Comment by squalus** [comment]
I reproduced the failure on master and the update PR should fix it.

**Comment by squalus** [comment]
Fixed by #172323
